F1WM.UnitTests/Services/SeasonServiceTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using F1WM.ApiModel;
+using F1WM.Repositories;
+using F1WM.Services;
+using Moq;
+using Xunit;
+
+namespace F1WM.UnitTests.Services
+{
+    public class SeasonServiceTests
+    {
+        private SeasonService service;
+        private Mock<ISeasonRepository> seasonRepositoryMock;
+        private Mock<ITimeService> timeServiceMock;
+
+        public SeasonServiceTests()
+        {
+            seasonRepositoryMock = new Mock<ISeasonRepository>();
+            timeServiceMock = new Mock<ITimeService>();
+            service = new SeasonService(seasonRepositoryMock.Object, timeServiceMock.Object);
+        }
+
+        [Fact]
+        public async Task ShouldGetSeasonRules()
```

Comment:
There are actually two paths to test:  
- get current season's rules when year is null
- get season's rules based on year parameter  

No big deal, but it would be nice to cover both. The same applies to CalendarService